admin/partials/acc_user_importer-admin-settings.php

```diff
@@ -45,6 +45,11 @@ function acc_email_settings() {
 		require_once (ACC_BASE_DIR . '/template/email_settings.php');
 	}
 
+	// Define functions to get default values from different files.
+	function accUM_get_login_name_mapping_default() {return 'Firstname Lastname';}
+	function accUM_get_update_user_login_default() {return 'No';}
+	function accUM_get_default_role_default() {return 'Subscriber';}
+
```

Comment:
This is so we can get the default values from 2 different files.  I think it looks ugly, but was not sure how to do better.